config/default.yml

```diff
@@ -1884,6 +1884,12 @@ Naming/PredicateName:
   Exclude:
     - 'spec/**/*'
 
+Naming/RescuedExceptionsVariableName:
+  Description: 'Use consistent rescued exceptions variables naming.'
+  Enabled: true
+  VersionAdded: '0.61'
+  PreferredName: exception
```

Comment:
I searched [ruby/ruby](https://github.com/ruby/ruby), [rubygems/rubygems](https://github.com/rubygems/rubygems), [bundler/bundler](https://github.com/bundler/bundler), and [rails/rails](https://github.com/rails/rails) (,and my local OSS and proprietary) repositories using `grep`. In many cases, variable name `e` was used as the local variable name of `rescue` clause.
Since `rescue` is a limited scope for handling exception, I think that naming context is obvious in variable name `e`. 
I think the default value is preferred to `PreferredName : e` rather than `PreferredName: exception` 😉

Reply:
I disagree, it's not because it's mostly used that it is a good thing. RuboCop is also there to educate the Rubyists. 
Moreover, the variable `e` is contrary to the Ruby Style Guide which clearly defines that a variable name must be explicit and make more than 3 characters. 'exception' is clearly a better choice.

Reply:
I don't think so. This context variable name is generally used with `e`, and I think that defaults should start with wide conventions. It would be good to have consistent names, but default name to `exception` would be uncomfortable.

> RuboCop is also there to educate the Rubyists.

I listened to some Ruby committers at RubyConf 2018 in LA, the result was that `e` was preferred. It also appears in the ruby/ruby repository code.
https://github.com/ruby/ruby

>  the variable `e` is contrary to the Ruby Style Guide which clearly defines that a variable name must be explicit and make more than 3 characters.

I will quote it again. Since `rescue` is a limited scope for handling exception, I think that naming context is obvious in variable name `e`.

There are some configured abbreviation names in Naming/UncommunicativeMethodParamName cop.

```yaml
  AllowedNames:
    - io
    - id
    - to
    - by
    - 'on'
    - in
    - at
    - ip
    - db
```

https://github.com/rubocop-hq/rubocop/blob/v0.60.0/config/default.yml#L1912-L1921

The `rescue` clause scope variable name `e` is a popular variable name that as well as these.

Reply:
I'm personally fine with the use of `e`/`ex` in such context, as I agree with @koic. I think the main purpose of this cop is to enforce consistency, not any particular naming scheme, so the only way I'd agree for this to be enabled by default is if the default is the common practice today.

Reply:
variable names should be descriptive, single char name can have many meanings, it doesn't matter what the scope of code is